### DISCUSSION TITLE
404 page title

### DISCUSSION THREAD

**Discussion by m-berger-nf** [statamic/cms#6343] [Q&A] [answered]
I am currently adjusting the 404 error page. How can I overwrite the default page title (which currently says "Statamic") without having to create a custom layout page that hardcodes the title tag?

Update: Realized that the "Statamic" is coming from the respective environment variable since I have this in the layout page:
`<title>{{ title ?? site:name }}</title>`.
If I change it to `<title>{{ title }}</title>` the resulting title tag is empty. Still not sure what's the right approach to set the title for the 404 page.

Thanks in advance.

**Comment by ryanmitchell** [comment]
Why not revert the change and modify the site name in the .env file?

**Reply by m-berger-nf** [reply]
Thanks, I guess that would work, as long as there is no other use case in which the title variable can be empty and the site name will be used then. I'd like to have the title say something like "Not found", so I have to be sure that it is only used when the 404 page is shown. I was hoping for some sort of best practice or setting specifically for the 404 page, that I might be missing.

**Reply by ryanmitchell** [reply]
You can set the title using section, and then yield it in your template.
See: https://statamic.dev/tags/section

**Comment by joshuablum** [comment, marked as answer]
Hey @m-berger-nf,

Like @ryanmitchell mentioned, you should ideally change the site name in the `.env` file.

But there are a few other options:
- Using the `{{ section }}` tag like Ryan suggested
- [Using a ViewModel](https://statamic.dev/view-models)
- [Using view front matter in the template](https://statamic.dev/variables/#view-frontmatter)
- [Using a custom layout for the error page](https://statamic.dev/routing#error-pages), like you mentioned yourself

I would suggest to have a look at [the docs on the cascade order of views](https://statamic.dev/variables/#view-frontmatter)

Hope that helps :)

**Comment by m-berger-nf** [comment]
Thanks @ryanmitchell and @joshuablum for the great input!

I have gone with the `{{ section }}` option for now. Will look into the other options as well. And certainly will read up more on the cascade.
